public/index.html

```diff
@@ -51,7 +52,8 @@
     <script src="auth.js"></script>
     <script src="data.js"></script>
     <script src="UI.js"></script>
-    <script src="graph.js"></script>
+    <script src="traffic_monitoring_graphs.js"></script>
+    <script src="systems_monitoring_graphs.js"></script>
```

Comment:
The line here is twice as thick as all the others:
<img width="176" alt="image" src="https://user-images.githubusercontent.com/7963608/83024035-a6268d80-a025-11ea-8edd-02f52d65cda7.png">
I think we had this problem before, so hopefully it's straightforward to apply the same solution again here

Reply:
Updated the right border position, thanks